Tests/Unit/MSFT_xADDomainController.Tests.ps1

```diff
@@ -56,17 +59,43 @@ try
 
     #region Function Get-TargetResource
     Describe -Tag 'xADDomainController' "$($Script:DSCResourceName)\Get-TargetResource" {
-        It 'Returns current "SiteName"' {
-            Mock Get-ADDomain { return $true } @commonMockParams
-            Mock Get-ADDomainController {
-                return $stubDomainController = @{
-                    Site = 'PresentSite'
-                    Domain = 'present.com'
-                }
-            } @commonMockParams
 
-            $result = Get-TargetResource @testDefaultParams -DomainName $correctDomainName -SiteName $correctSiteName
+        Mock Get-ADDomain { return $true } @commonMockParams
+        Mock Get-ADDomainController {
+            return $stubDomainController = @{
+                Site = 'PresentSite'
+                Domain = 'present.com'
+            }
+        } @commonMockParams
+        Mock Get-ItemProperty -ParameterFilter { $Path -eq 'HKLM:\SYSTEM\CurrentControlSet\Services\NTDS\Parameters' } {
+            return @{
+                'Database log files path' = 'C:\Windows\NTDS'
```

Comment:
>C [](start = 45, length = 1)

Same here. You have hard coded path in multiple places please review all of them.